# error.py
print(1/0)